.circleci/config.yml

```diff
@@ -21,22 +21,11 @@ jobs:
       
       - checkout:
           name: Checking out the code
-      # Download and cache dependencies
-      - restore_cache:
-          name: Restore Yarn Package Cache
-          keys:
-            - yarn-packages-{{ checksum "yarn.lock" }}
```

Comment:
Would this change mean that our build will become slower because it will reinstall all dependencies for each build?

Reply:
@bogdan they won't slowdown, currently our cache restoring doesn't work at all :P

I'm seeing this in every CI build:
```
error computing cache key: template: cacheKey:1:17: executing "cacheKey" at <checksum "yarn.lock">: error calling checksum: open /home/circleci/project/yarn.lock: no such file or directory
```

The thing is our `yarn.lock` is gitignored (and it's good). I dunno what are the approaches for caching library dependencies for CI.. Maybe `checksum "package.json"` will do the trick?

Reply:
I believe we indeed could do something like this:
```
keys:
 - npm-deps-{{ .Branch }}
 - npm-deps-{{ checksum "package.json" }}
 - npm-deps-
```

The caches are restored from the first matched key, so I think this is a sane fallback strategy: branch -> package.json -> the most recent cache from any branch